inductiva/tasks/task.py

```diff
@@ -216,19 +206,32 @@ def get_output(
         """
         self.wait()
 
-        # Check if default files are set by the output class
+        # Check if already have the info and the task is in a terminal status.
+        if self._info is not None and self._status in _TASK_TERMINAL_STATUSES:
```

Comment:
Why do you need this if statement, get_info() is already caching results inside with a similar condition, no? Can't you just always call get_info() here?

Reply:
Indeed, sorry for this!